### PR TITLE
Remove noMainMedia to make way for PrintShop

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?

Co-authored with @oliverlloyd

Remove `noMainMedia` and all `noMainMedia` checks and instead look for the the new design PrintShop, a design which is not designed to have a main media element. 

For now `decideDesignType` has had tags added as a prop to check if a tag indicates a design is PrintShop.

`if (
		tags.find((tag) => tag.id === 'artanddesign/series/guardian-print-shop')
	) {
		return Design.PrintShop;
	}
`

A story has also been added to confirm it appears as expected.

Screenshot 2021-01-22 at 14 27 39

### CODE REVIEW

**Review by shtukas — APPROVED** [review]

**Review by liywjl — APPROVED** [review]
LGTM

File: src/web/components/ArticleTitle.tsx [hunk 1]
```diff
@@ -69,32 +68,34 @@ export const ArticleTitle = ({
...
-			</div>
-		)}
-		<div
-			className={cx(
-				badge && marginTop,
-				display === Display.Immersive &&
-					!noMainMedia &&
-					immersiveMargins,
+}: Props) => {
+	return (
```
Comment (oliverlloyd) [inline src/web/components/ArticleTitle.tsx:72]: If you restore this to `}: Props) => (` the diffs will align better!

File: src/web/components/ArticleTitle.tsx [hunk 2]
```diff
@@ -69,32 +68,34 @@ export const ArticleTitle = ({
...
-				sectionLabel={sectionLabel}
-				sectionUrl={sectionUrl}
-				guardianBaseURL={guardianBaseURL}
-				pillar={pillar}
-				badge={badge}
-			/>
+			<div
+				className={cx(
+					badge && marginTop,
+					design !== Design.PrintShop && immersiveMargins,
```
Comment (oliverlloyd) [inline src/web/components/ArticleTitle.tsx:85]: Oh, I see what we did! It should be 

```
display === Display.Immersive && design !== Design.PrintShop && immersiveMargins
```

**Comment by oliverlloyd** [comment]
[Use PrintShop design to replace noMainMedia](https://trello.com/c/Bt0WqGch/2366-use-printshop-design-to-replace-nomainmedia)

**Pull request merged**